Makefile

```diff
@@ -111,6 +112,21 @@ build-query-linux:
 build-collector-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/collector/collector-linux ./cmd/collector/main.go
 
+.PHONY: docker
+docker: build_ui build-agent-linux build-collector-linux build-query-linux
+	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
+	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema plugin/storage/cassandra/ ; \
```

Comment:
I talked to @jsanda last week about something related to this and he made me realize that this might also not be appropriate.

In any case: I think we could add a separate task for having a more reliable bootstrap method, perhaps consistent among all deployment architectures. In the meantime, I'd keep it like this, so that we can move forward.